docs/LINUX-DIST.md

```diff
@@ -50,3 +50,10 @@ nix-env -i lpac
 ```
 
 see <https://github.com/nix-community/nur-combined/blob/master/repos/linyinfeng/pkgs/lpac/default.nix>
+
+## Ubuntu and Debian/Devuan
+
+> Minimum available release: 14.04 Trusty Tahr
```

Comment:
```suggestion
> Minimum available release: [14.04 Trusty Tahr](https://releases.ubuntu.com/14.04/)
```

Reply:
LGTM. The reason I thought it OK to leave out was that it's left out in the OpenWrt section.

Reply:
Snapshot not is General available

Reply:
Debian codename and Ubuntu codename is inconsistent

I think should separate

Reply:
> OpenWrt Snapshot are not generally available

OpenWrt snapshot is the counterpart of Debian Sid. Debian Sid is very available.

Reply:
> Debian codename and Ubuntu codename is inconsistent
> 
> I think should separate

The release schedules were always different. Trusty Tahr is from more than 10 years ago, and everyone will be using something far more recent. The package I made should work the same on every version of both since then.